services/web/server/tests/unit/with_dbs/test_change_password.py

```diff
@@ -15,7 +15,7 @@
 NEW_PASSWORD = "NewPassword1*&^"
 
 
-async def test_unauthorized(client):
+async def test_unauthorized(loop, mock_orphaned_services, client):
```

Comment:
It is fine but why not add ``mock_orphaned_services`` fixture to the ``client`` fixture instead of to every test? In a way what you want is to "disable" this function from the service, right? this is then less noisy to the tests if you set it up at the client fixture

Reply:
I will give this a try. I had not thought about the possibility